tdrs-backend/tdpservice/search_indexes/documents.py

```diff
@@ -0,0 +1,375 @@
+
+        model = T2
+        fields = [
+            'record',
+            'rpt_month_year',
+            'case_number',
+            'disposition',
+            'fips_code',
+
+            'family_affiliation',
+            'noncustodial_parent',
+            'date_of_birth',
+            'ssn',
+            'race_hispanic',
+            'race_amer_indian',
+            'race_asian',
+            'race_black',
+            'race_hawaiian',
+            'race_white',
+            'gender',
+            'fed_oasdi_program',
+            'fed_disability_status',
+            'disabled_title_xivapdt',
+            'aid_aged_blind',
+            'receive_ssi',
+            'marital_status',
+            'relationship_hoh',
+            'parent_minor_child',
+            'needs_pregnant_woman',
+            'education_level',
+            'citizenship_status',
+            'cooperation_child_support',
+            'months_fed_time_limit',
+            'months_state_time_limit',
+            'current_month_state_exempt',
+            'employment_status',
+            'work_eligible_indicator',
+            'work_part_status',
+            'unsub_employment',
+            'sub_private_employment',
+            'sub_public_employment',
+            'work_experience_hop',
+            'work_experience_ea',
+            'work_experience_hol',
+            'ojt',
+            'job_search_hop',
+            'job_search_ea',
+            'job_search_hol',
+            'comm_services_hop',
+            'comm_services_ea',
+            'comm_services_hol',
+            'vocational_ed_training_hop',
+            'vocational_ed_training_ea',
+            'vocational_ed_training_hol',
+            'job_skills_training_hop',
+            'job_skills_training_ea',
+            'job_skills_training_hol',
+            'ed_no_high_school_dipl_hop',
+            'ed_no_high_school_dipl_ea',
+            'ed_no_high_school_dipl_hol',
+            'school_attendence_hop',
+            'school_attendence_ea',
+            'school_attendence_hol',
+            'provide_cc_hop',
+            'provide_cc_ea',
+            'provide_cc_hol',
+            'other_work_activities',
+            'deemed_hours_for_overall',
+            'deemed_hours_for_two_parent',
+            'earned_income',
+            'unearned_income_tax_credit',
+            'unearned_social_security',
+            'unearned_ssi',
+            'unearned_workers_comp',
+            'other_unearned_income',
+        ]
+
+
+@registry.register_document
+class T3DataSubmissionDocument(Document):
+    """Elastic search model mapping for a parsed T3 data file."""
+
+    class Index:
+        """ElasticSearch index generation settings."""
+
+        name = 't3_submissions'
+        settings = {
+            'number_of_shards': 1,
+            'number_of_replicas': 0,
+        }
+
+    class Django:
+        """Django model reference and field mapping."""
+
+        model = T3
```

Comment:
@jtimpe another note for awareness:

T3 is a person-level record that holds data for up to 2 people. This is also true for M3 records (SSP-MOE Section 1, children's data)

Ultimately, we want the data to be stored in the fields as it appears herein, but we may need an interim step to parse the whole record, then restructure the data before it can be stored. (for reference, see cells 26 and 27 in the parsing notebook)